torch/_inductor/compile_fx.py

```diff
@@ -844,11 +844,19 @@ def _compile_fx_inner(
 
     log.debug("FX codegen and compilation took %.3fs", time.time() - start)
 
-    # This message is for printing overview information of inductor mm counts, shapes,etc after lowering
-    if log.isEnabledFor(logging.INFO):
+    # This message is for printing overview information of inductor mm counts, shapes, etc after lowering
+    if log.isEnabledFor(logging.INFO) and len(counters["aten_mm_info"].items()):
```

Comment:
No need for item call? Check if the keys are empty is sufficient?

Reply:
+1